### ISSUE TITLE
Can't migrate to an older version (121 > 12).

### ISSUE THREAD

**Issue by soixt** [unlayer/vue-email-editor#84]
When I load existing design into editor I keep getting this:
Can't migrate to an older version (121 > 12).

**Comment by brunolemos** [comment]
The `schemaVersion` field in your json should be a number, not a string. Please also check all other values like numbers and booleans to make sure they also didn't become a string.

**Comment by soixt** [comment]
Yeah, I figured it was something like this when I manually changed version to number. Can database type for design be json or text?
